docs/src/probabilities.md

```diff
@@ -1,10 +1,19 @@
-# Probabilities
+# [Probabilities](@ref probabilities)
 
 !!! note
-    Please be sure you have read the [Terminology](@ref terminology) section before going through the API here.
+    Please be sure you have read the [Terminology](@ref terminology) section before going through the API here to have a good idea of the terminology used in ComplexityMeasures.jl.
```

Comment:
The "terminology" link no longer works. Which section, specifically, is the used supposed to read first? The link needs updating.